File: bindings/dart/example/upbit_provider.dart
import 'package:maxt/maxt.dart';

Future<void> main() async {
  await Maxt.initialize();
  try {
    final adapter = UpbitAdapter();
    final market = Market.spot(Exchange.upbit, 'BTC', 'KRW');
    final tickers = await adapter.tickers([market]);
    final instruments = await adapter.orderbookInstruments([market]);
    print('region=${adapter.region}; ticker rows=${tickers.length}');
    print('order-book instrument rows=${instruments.length}');
  } finally {
    await Maxt.dispose();
  }
}

File: bindings/python/python/maxt/examples/upbit_provider.py
"""Read Upbit Korea provider-specific quotation data."""

from __future__ import annotations

import asyncio

from maxt import Exchange, Market, UpbitAdapter


async def main() -> None:
    adapter = UpbitAdapter()
    market = Market.spot(Exchange.UPBIT, "BTC", "KRW")
    tickers = await adapter.tickers([market])
    instruments = await adapter.orderbook_instruments([market])
    print(f"region={adapter.region}; ticker rows={len(tickers)}")
    print(f"order-book instrument rows={len(instruments)}")


if __name__ == "__main__":
    asyncio.run(main())
